servant/src/Servant/API.hs

```diff
@@ -83,6 +87,7 @@ import           Servant.API.ResponseHeaders (AddHeader (addHeader),
                                               HList (..), Headers (..),
                                               getHeadersHList, getResponse)
 import           Servant.API.Sub             ((:>))
+import           Servant.API.Times           (FTime(..), toProxy, getFormat, renderTime, parseTime)
```

Comment:
I'm inclined to say we should remove it. It's available in `Times.hs` if really needed.